lib/rubocop/cop/ezcater/rspec_require_http_status_matcher.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Ezcater
+      # Enforce use of HTTP status code matchers rather than asserting on on random numbers.
+      #
+      # @example
+      #
+      #   # good
+      #   expect(response).to be_created
+      #   expect(response).to be_bad_request
+      #
+      #   # bad
+      #   expect(response).to have_http_status 201
+      #   expect(response).to have_http_status 400
+      class RspecRequireHttpStatusMatcher < Cop
+        MSG = "Use the `have_http_status` matcher, like `expect(response).to have_http_status :bad_request`, "\
```

Comment:
Are the good and bad examples above reversed? This is recommended what is listed as "bad" above?